src/components/login/index.jsx

```diff
@@ -0,0 +1,78 @@
+import { Input } from "antd";
+import React, { useState } from "react";
+import "./index.css";
+
+const Login = (props) => {
+  const [values, setValues] = useState({ email: "", password: "" });
+
+  const handleChange = (key, value) => {
+    let newValues = Object.assign({}, values);
+    newValues[key] = value;
+    setValues(newValues);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    if (typeof props.onSubmit == "function") {
+      props.onSubmit(values);
+    }
+  };
+  const handleGoToRegister = () => {
+    if (typeof props.onGoToRegister == "function") {
+      props.onGoToRegister();
```

Comment:
I am using the main file (components/LoginRegister) for state management regarding conditional rendering, this will be much clearer when the final PR is submitted.